cartoframes/data/services/bq_datasets.py

```diff
@@ -291,11 +291,13 @@ def import_dataset(self):
     def upload_dataframe(self, dataframe):
         return self._client.upload_dataframe(dataframe, self._name)
 
-    def enrichment(self, geom_type='points', variables=None, output_name=None):
+    def enrichment(self, geom_type='points', variables=None, filters=None, aggregation=None, output_name=None):
```

Comment:
I'd use empty collections instead of `None` as defaults where it applies.

Reply:
I had issues with mutable default values for arguments: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Not sure if it applies here

Reply:
Followed @oleurud tips!